src/pwr.rs

```diff
@@ -427,6 +431,14 @@ impl Pwr {
             vos = VoltageScale::Scale0;
         }
 
+        // 725, 735, and the like don't have the overdrive bit,
+        // and vos0 can be switched to just like vos1-3.
+        #[cfg(all(feature = "revision_v", feature = "rm0468"))]
+        if self.enable_vos0 {
+            vos = VoltageScale::Scale0;
+            self.voltage_scaling_transition(vos);
```

Comment:
Good catch! I find the documentation for these things to be... formidable.

That paragraph seems to also imply that ACTVOSRDY must be checked _after_ the VOS change. Other parts of the documentation (section 6.4.1, at the bottom of page 238) say that ACTVOSRDY must be checked _before_ the VOS change (which the pwr.rs code already does). Basically it looks like there should also be another `while self.rb.csr1.read().actvosrdy().bit_is_clear() {}` loop after voltage_scaling_transition... Does this sound useful at all? Or am I misinterpreting the specs?

Reply:
Agreed, at this level of detail the Reference Manuals can be quite unclear. I think section 6.4.1 is about the transition before entering Run mode, whilst the transition we are interested in here is entering VOS0 (from VOS3). So these are two separate transitions, at least according to the reference manual. 

I'd agree with your interpretation that ACTVOSRDY should also be checked when entering VOS0. From a quick check on the STM32H735G-DK board, it seems that ACTVOSRDY is already clear after the transition to enter VOS0 (just like VOSRDY == ACTVOSRDY is already true after the same). So it doesn't seem to make much difference, but I'd still support adding the check.

Reply:
Great! I added the two busy loops to check for VOS==ACTVOS and ACTVOSRDY=true. I also updated the imports since my changes don't use RCC (this was causing a warning).